packages/core/src/i18n/translations/en.json

```diff
@@ -158,6 +158,8 @@
   "Base_Dashboard": "Dashboard",
   "Base_Dashboard_Misconfigured": "This dashboard is misconfigured, please contact your administrator.",
   "Base_Dashboard_RefreshConfig": "Refresh dashboard",
+  "Base_Loader_proccessSuccessMessage": "Process successfully completed.",
+  "Base_Loader_proccessErrorMessage": "An error has occurred!",
```

Comment:
Caution to capital letter in key: Base_Loader_ProccessSuccessMessage and Base_Loader_ProccessErrorMessage

Also, remove the exclamation point at the end of the error message, it's a bit harsh